web/src/router.rs

```diff
@@ -172,8 +184,12 @@ fn note_routes(app_state: AppState) -> Router {
     Router::new()
         .route("/notes", post(note_controller::create))
         .route("/notes/:id", put(note_controller::update))
-        .route("/notes", get(note_controller::index))
-        .route_layer(from_fn_with_state(app_state.clone(), protect::notes::index))
+        .merge(
+            // GET /notes
```

Comment:
Adding these inline for readability as there are a few nestings of functions/methods